plugins/common/functions

```diff
@@ -549,6 +549,8 @@ dokku_deploy_cmd() {
         dokku_log_info2 "stopping $APP.$PROC_TYPE ($cid)"
         # shellcheck disable=SC2086
         docker stop $DOCKER_STOP_TIME_ARG "$cid" &> /dev/null
+        # remove cid from oldids to skip the old container finish processing
+        oldids="$(remove_val_from_list "$cid" "$oldids")"
```

Comment:
The default delimiter in `remove_val_from_list` is a comma. However`oldids` is space-separated. I believe you will need to change this call to this
```
oldids="$(remove_val_from_list "$cid" "$oldids" " ")"
```